openquake/hazardlib/gsim/projects/acme_2019.py

```diff
@@ -197,8 +197,30 @@ class AlAtikSigmaModel(GMPE):
     in the current implementation of the NGA East GMMs.
 
     :param GMPE:
-        GMPE is the **kwargs describing the GMPE (its name) and the sigma 
+        GMPE is the **kwargs describing the GMPE (its name) and the sigma
         and kappa modifications to be imposed
```

Comment:
Where did you find the syntax `:kwarg` ?

Reply:
I was simply replacing the word "param" since this was not deemed appropriate for the description of gmpe_name. Is there a different term that should be used?

Reply:
Unless there is a specific sphinx syntax for kwargs (you should study sphinx) I would keep using `:param`.